packages/dashboard/src/explorer/components/NodeUsedResources.vue

```diff
@@ -43,7 +43,7 @@
     </v-row>
     <v-row justify="center">
       <v-progress-circular v-if="loader" indeterminate color="primary" :size="50" class="mt-10 mb-10" />
-      <v-btn v-if="!loader" class="mt-5 primary" @click="openGrafanaUrl">Monitoring url</v-btn>
+      <v-btn v-if="nodeStatus && !loader" class="mt-5 primary" @click="openGrafanaUrl">Monitoring url</v-btn>
```

Comment:
As a user i'd be interested going into the dashboard of the node up/down doesn't matter, if it's down i can still see the previous info